.github/workflows/release-prod.yml

```diff
@@ -6,12 +6,25 @@ on:
       - 'v*'
 
 jobs:
+  check-master:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check master branch
+        id: check
+        run: |
+          SHA=$(git log -n1 --format=format:"%H")
+          git checkout master
+          git merge-base --is-ancestor $SHA HEAD
```

Comment:
I think we need to return non-zero exit code in case `git merge-base --is-ancestor $SHA HEAD` return 0.

Reply:
> I think we need to return non-zero exit code in case `git merge-base --is-ancestor $SHA HEAD` return 0.

if the branch is master, it returns `0` denoting a successful exit and the next job continues

Reply:
My bad, just this as last line `git merge-base --is-ancestor $SHA HEAD` is correct.